sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -6817,8 +6817,8 @@ private void WithAttributes(NamedDecl namedDecl, bool onlySupportedOSPlatform =
 
                     case CX_AttrKind_Annotate:
                     {
-                        var annotationText = attr.Spelling;
-                        outputBuilder.WriteCustomAttribute($"""NativeAnnotation("{annotationText}")""");
+                        var annotationText = attr.Spelling.Replace("\"", "\"\"", StringComparison.Ordinal);
```

Comment:
This should rather use the `EscapeString(string value)` helper which centralizes all the escape string handling and attempts to handle already escaped sequences as well.

Reply:
Done